### DISCUSSION TITLE
Errore modulo evacuazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#666] [Richieste di aiuto]
Buongiorno,non appena mi reco per compilare il modulo di evacuazione mi compare l'errore 500

allego 
[2024-03-16_17-10.log](https://github.com/iisgiua/giuaschool/files/14624322/2024-03-16_17-10.log)

**Comment by trinko** [comment]
Il log riportato è relativo alle sole pagine di login e di Sistema>Manutenzione... 
Non contiene quindi i dati della pagina che riporta l'errore indicato.

Comunque, la cosa più probabile è che ci sia un errore nel template nei nomi dei campi: devono essere esattamente gli stessi nomi usati nella definizione del modulo.
